dashboard/config/scripts/commute_assess_comp_2_block.multi

```diff
@@ -1,5 +1,6 @@
 name 'Commute Assess - comp 2 - block'
-question 'The procedure <span style="font-family: monospace">op4</span> takes in 3 numbers, what does this procedure do?'
+editor_experiment ''
```

Comment:
Probably not an issue but wanted to call out this was added on this level but not the ones below which are the same level just for different years

Reply:
this looks like my bug -- editor_experiment shouldn't be getting added when the value is nil/empty. not sure what could be causing the difference though

Reply:
the difference is just that I edited the first one in my local levelbuilder interface so I could verify the results, then copied the changes into the other files using my text editor